dll/win32/newdev/wizard.c

```diff
@@ -567,6 +567,34 @@ AutoDriver(HWND Dlg, BOOL Enabled)
     IncludePath(Dlg, Enabled & IsDlgButtonChecked(Dlg, IDC_CHECK_PATH));
 }
 
+static INT CALLBACK
+BrowseCallbackProc(HWND hwnd, UINT uMsg, LPARAM lParam, LPARAM lpData)
+{
+    switch (uMsg)
+    {
+        case BFFM_INITIALIZED:
+        {
+            SendMessage(hwnd, BFFM_SETSELECTION, TRUE, lpData);
+            SendMessage(hwnd, BFFM_ENABLEOK, 0, FALSE);
+            break;
+        }
+
+        case BFFM_SELCHANGED:
+        {
+            WCHAR szDir[MAX_PATH];
+            if (SHGetPathFromIDList((LPITEMIDLIST)lParam, szDir))
+            {
+                 PDEVINSTDATA DevInstData = (PDEVINSTDATA)lpData;
+                 PrepareFoldersToScan(DevInstData, FALSE, TRUE, GetDlgItem(GetParent(hwnd), IDC_COMBO_PATH));
```

Comment:
Why is this `PrepareFoldersToScan` required?
What should be done if there was another directory selected again?